src/scancode/cli.py

```diff
@@ -231,11 +231,46 @@ class ScanCommand(BaseCommand):
     short_usage_help = '''
 Try 'scancode --help' for help on options and arguments.'''
 
-    def get_params(self, ctx):
-        """
-        Add options returned by plugins to the params list
-        """
-        return super(BaseCommand, self).get_params(ctx)
+    def format_options(self, ctx, formatter):
+        """Writes all the option groups into the formatter if they exist."""
```

Comment:
Can you use the convention we use else for docstrings ? e.g. on multiple lines and using imperative style (Write and not Writes)